### ISSUE TITLE
'︃@' = 'at' not transferred correctly

### ISSUE THREAD

**Issue by dieter-erich** [TigerVNC/tigervnc#1823]
I run the UltraVNC server 1.4.0.6 on a Windows7 machine and connect to it from an Ubuntu box running 24.04, 20.04 or any other version by using the most recent RealVNC viewer or the TurboVNC v3.0.1 viewer with the same results:

Everything works perfectly well except from "︃︃@" not being transferred correctly. When typing it, it appears as "︃@" on the screen but when using it e.g. within an e-mail address I receive the message "enter a correct e-mail address" or something alike. So, despite being shown as the correct alphanumerical sign it is apparently not recognized as such internally. Answering an e-mail works without problems as long as the address is already present in the "TO" field; i.e. replying works seamlessly. However, even copying '︃@' from the local PC to the remote PC does not work. The same problem occurs in browser windows. 

For many years I used the RealVNC server without problems. I also used Anydesk without problems. The problem occurred only when going back to the above free VNC servers/clients. I tried several updates and also tried older versions to no avail. 

In summary various clients work in combination with RealVNC but do not with the UltraVNC server. Anydesk works seamlessly. Please note that I need to stay with Windows7 and do not want to buy a license for Anydesk or RealVNC. 
The programs were downloaded from the respective developer sites
 Help is greatly appreciated!

**Comment by bphinz** [comment]
Have you tried using the TigerVNC client? The description you’ve provided doesn’t reference our products at all.

**Comment by dieter-erich** [comment]
It seems that the problem occurs with whatever client, which strongly suggests that it is related to the server. As I wrote earlier, it did not occur with the RealVNC server but does so with the UltraVNC server! Is there any other free server I might try?

**Comment by CendioOssman** [comment]
TightVNC has one as well, I think. Technically, we do as well, but it is unmaintained and has known problems.

Since this doesn't seem to be a TigerVNC issue, I'm going to go ahead and close this issue.
